DataSetExplorer/Program.cs

```diff
@@ -81,35 +92,25 @@ private static ListDictionary GetAnnotatedProjects()
         private static void CheckAnnotationConsistencyForAnnotator(int annotatorId)
         {
             var instancesGroupedBySmells = GetAnnotatedInstancesGroupedBySmells(annotatorId);
+            var manovaTest = new AnnotationConsistencyTests.ManovaTest();
 
-            var exporter = new AnnotationConsistencyByMetricsExporter("D:/ccadet/annotations/sanity_check/Output/");
-            var manovaTest = new ManovaTest.ManovaTest();
-
-            var enumerator = instancesGroupedBySmells.GetEnumerator();
-            while (enumerator.MoveNext())
+            foreach (var codeSmellGroup in instancesGroupedBySmells)
```

Comment:
You have 3 repetitions of this code, where only the final command is different. Can you come up with an alternative design?

This Program.cs class will require major refactoring anyway so it is not that important that we solve it right now, but contemplate this issue a bit to see if you can come up with a better design.